### ISSUE TITLE
admin UI - field type email and validation

### ISSUE THREAD

**Issue by Jimmi08** [e107inc/e107#3824]
My settings:

```
			'survey_mailto' =>   array ( 'title' => ADLAN_SUR15,
				'type' => 'email',
				'data' => 'str',
				'width' => 'auto',
				'help' => '',
				'validate' => true,
				'readParms' => '',
				'writeParms' => '',
				'class' => 'left',
				'thclass' => 'left',
				), 
```

because Chrome autofilling is filled with the user name, so validation is always true. 

But why there is not at least check if it's email address?   What is the difference between the text field then?  Thanks

PHP  7.0

**Comment by Moc** [comment]
This is a bug but not sure what is causing it. 

The Admin UI renders correctly `<input type="email"> but the browser (in my case Chrome) does not seem to validate at all. No message is displayed.

@CaMer0n @SimSync Any idea?

**Comment by Jimmi08** [comment]
Maybe it doesn't related to email, just to any text fill. See this: 
 
https://github.com/e107inc/e107/issues/3131

**Comment by Moc** [comment]
No, I don't think it's related to autocomplete. For some reason the built in browser validation is not triggered.
